### DISCUSSION TITLE
update process

### DISCUSSION THREAD

**Discussion by cryptophangan** [Chia-Network/chia-blockchain#18270] [Support]
I have a question regarding the update process. If I upgrade my old version of Chia to the new version, will the new application prompt me to enter my seed phrases after installation? Windows.

Thank you.

**Comment by cryptophangan** [comment]
Can you answer please... i dont know what to do

**Comment by BrandtH22** [comment]
Hey @cryptophangan , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

As for upgrading chia on windows I generally:
- stop chia
- uninstall the current version of chia
- install the new version of chia
- start chia

This process will not require you to re-enter the seed phrase as those keys are stored in the `~/.chia_keys` directory which is not touch in the process. 

Note that when upgrading major versions of chia you may need to have a new config file created by renaming the config file located at `~/.chia/mainnet/config/config.yaml` after uninstalling the old version and prior to installing the new version. When running chia for the first time on the new version a new config will be created then you can update the config as needed (resync the wallet using the GUI settings / advanced / resync option to have any pool information readded, use the farm tab to readd plot directories, etc and we can help with this process if it is needed).

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
